config/rules/ccnumber.yaml

```diff
@@ -0,0 +1,14 @@
+---
+Searcharea: body
+rules:
+  - Code: 2001
+    Pattern: "[^\\.](\\b3[47]\\d{13}\\b)"
+    Caption: Potential American Express credit card number in file
```

Comment:
Only Amex rules?

Reply:
Yes, according to @grinish21 we should only be open sourcing AMEX

Reply:
I don't think so. @grinish21 can we discuss?